src/Microsoft.OData.Client/DeleteLinkUriOption.cs

```diff
@@ -0,0 +1,26 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="DeleteLinkUriOption.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Client
+{
+    /// <summary>
+    /// Used to specify the form of Uri to be used for a delete link request.
+    /// </summary>
+    public enum DeleteLinkUriOption

```

Comment:
Out of curiosity, does this issue with different URI options only affect DELETE requests?